client/bridge_client.go

```diff
@@ -172,6 +172,34 @@ func (ec *Client) BridgeGetSentChainTxsLimit(ctx context.Context) (uint64, error
 	return result, err
 }
 
+// BridgeRegisteredOperators registers an operator.
+func (ec *Client) BridgeRegisterOperator(ctx context.Context, bridge, operator common.Address) (common.Hash, error) {
```

Comment:
```suggestion
func (ec *Client) BridgeRegisterOperator(ctx context.Context, bridge, operator common.Address) (result common.Hash, err error) {
```

Reply:
This PR is already applied to Dev branch.
I will consider this review next time.